modules/openglqt/CMakeLists.txt

```diff
@@ -58,6 +61,9 @@ target_link_libraries(inviwo-module-openglqt PRIVATE
     Qt${QT_VERSION_MAJOR}::Svg
     Qt${QT_VERSION_MAJOR}::OpenGLWidgets
 )
+if(QT_VERSION_MAJOR EQUAL 6)
+    target_link_libraries(inviwo-module-openglqt PRIVATE Qt6::OpenGLWidgets)
```

Comment:
I don't understand. Is this not equivalent to Qt${QT_VERSION_MAJOR}::OpenGLWidgets above?

Reply:
the one above should not be there...

Reply:
it is gone in the latest version